buildtest.sh

```diff
@@ -59,7 +59,7 @@ echo -e "\n\n\n\n\033[33;1mmaking dist tarball\033[m"
 mkdir build_dist
 cd build_dist
 ../source/configure
-make distdir=sdist dist-gzip
+${MAKE:-make} distdir=sdist dist-gzip
```

Comment:
I'd say we should then assign a new variable and reuse it everywhere that uses `make`, not only in this line.

Reply:
Done.